### PR TITLE
Fix pancakeswap bsc nft marketplace tables' blockchain misclassification

### PR DESCRIPTION
**Pull request by chef-seaweed**
Brief comments on the purpose of your changes:

**For fixing wrong blockchain **

I've checked that:

### General checks:
* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [ ] `coin_id` represents the ID of the coin on coinpaprika.com
* [ ] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

File: models/pancakeswap/bnb/pancakeswap_bnb_nft_fees.sql [hunk 1]
```diff
@@ -1,7 +1,7 @@
  {{
   config(
         alias='fees',
```
Comment (henrystats) [inline models/pancakeswap/bnb/pancakeswap_bnb_nft_fees.sql:3]:
```suggestion
        alias='nft_fees',
```

**Review by 0xRobin — APPROVED** [review]

**Review by jeff-dude — APPROVED** [review]
please note @chef-seaweed -- since the spells are being renamed, the old ones might persist in the database/data explorer. can you tag me here if they do?
i.e. `pancakeswap_bnb.events` and `pancakeswap_bnb.nft_events` may both exist

**Comment by chef-seaweed** [comment]
hi @jeff-dude I think `pancakeswap_bnb.events` still exists in the database/data explorer.

**Comment by jeff-dude** [comment]
> hi @jeff-dude I think `pancakeswap_bnb.events` still exists in the database/data explorer.

okay, i figured -- thanks for heads up. i will look into cleaning that up tomorrow

**Pull request merged**